src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -1947,5 +1947,29 @@ void M()
 
             Test(markup, expectedOrderedItems);
         }
+
+        [WorkItem(4144, "https://github.com/dotnet/roslyn/issues/4144")]
+        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        public void TestInaccessibleItem()
```

Comment:
Nitpick: name the test for what it's testing about inaccessible items, since without looking I could imagine this would either assert they are there or they aren't there.